docs/key_modules/models.md

```diff
@@ -218,6 +219,63 @@ assistant_response = agent.step(user_msg)
 print(assistant_response.msg.content)
 ```
 
+### 4.3 Using NVIDIA Models
+
+NVIDIA provides powerful language models through their API. Here's how to use NVIDIA models with CAMEL:
+
+1. First, obtain your API key from [NVIDIA AI Playground](https://build.nvidia.com/explore/discover).
+
```

Comment:
It's great to consider the documentation aspect, but I believe adding the model to the model list should suffice. Including each platform in the documentation might be too detailed